prog/kube-utils/main.go

```diff
@@ -15,14 +15,15 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/vishvananda/netlink"
+	weaveapi "github.com/weaveworks/weave/api"
+	"github.com/weaveworks/weave/common"
+	"golang.org/x/sys/unix"
```

Comment:
We have a bit of a convention where imports are split into three blocks: first Go standard library, then imports from outside the repo, then imports from inside the repo.